packages/vue/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "types": [],
     "typeRoots": ["./node_modules/@types"],
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": ["src/*"],
+      "#/types": ["src/utils/types"]
```

Comment:
nice

Reply:
This was accidentally pushed but a small spoiler something cool is coming.